BUILD.md

```diff
@@ -84,6 +84,7 @@ For other system requirements and other dependencies, please see [this section](
 |**Build Shared Library**|--build_shared_lib||
 |**Build Python wheel**|--build_wheel||
 |**Build C# and C packages**|--build_csharp||
+|**Build WindowsML**|--use_winml<br>--use_dml<br>--build_shared_lib|WindowsML depends on DirectML and the OnnxRuntime shared library.|
```

Comment:
Can we avoid users from typing --build_shared_lib and just infer it when --use_winml is specified? It's just one less thing to remember for users.